tests/queries/1_stateful/00184_allow_kafka_use_default_column.sql

```diff
@@ -0,0 +1,6 @@
+SET allow_kafka_use_default_column = 1;
+CREATE TABLE queue1 (
+                        timestamp UInt64 default 0,
+                        level String,
+                        message String
+) ENGINE = Kafka('localhost:9092', 'topic', 'group1', 'JSONEachRow');
```

Comment:
We need a test that makes sure the default values are properly generated. You need to add an integration tests, as for these tests we don't start kafka. Please make sure you write a test for all columns types that you allow.

Reply:
OK，Let me add that